source/develop/release-management/features/network/managed_mtu_for_vm_networks.html.md

```diff
@@ -120,9 +120,10 @@ Beside the possibility to specify the machine type individually for each VM,
 a default machine type for clusters of a certain level is configured.
 In cluster level 4.3 machine type rhel7.4.0 or later will be configured by
 default.
-If all hosts in all of your clusters of cluster level 4.2 supports the
-rhel7.4.0 machine type, the default machine type can be configured to rhel7.4.0
-using the following command to use the MTU feature already in oVirt 4.2:
+The usage of machine type rhel7.4.0 for new clusters of cluster level 4.2 or
+clusters which will be upgraded to cluster level 4.2 can be enabled already in
+oVirt 4.2 if all hosts in all of your clusters of cluster level 4.2 supports the
```

Comment:
"if all hosts" I don't think there is a question about it. ovirt-4.2 is based on el7.5.
Can we say something simpler: Users can opt-in for this feature by configuring a default machine type of rhel7.4.0 for 4.2 clusters. They can also opt-in per VM, by selecting rhel7.4.0 machine type of an individual VM.

Reply:
I tried to update using this words.